install.sh

```diff
@@ -84,7 +84,7 @@ initDownloadTool() {
 checkLatestVersion() {
   # Use the GitHub releases webpage to find the latest version for this project
   # so we don't get rate-limited.
-  CHECKLATESTVERSION_REGEX="[0-9][A-Za-z0-9\.-]*"
+  CHECKLATESTVERSION_REGEX="v?[0-9][A-Za-z0-9\.-]*"
```

Comment:
```suggestion
  CHECKLATESTVERSION_REGEX="v[0-9][A-Za-z0-9\.-]*"
```
Since all the upcoming releases after this one will be with the `v` prefix I think we should drop the `?`.